packages/coil-extension/manifest.json

```diff
@@ -23,6 +23,13 @@
   "web_accessible_resources": [
 
   ],
+  "declarative_net_request": {
+    "rule_resources": [{
+      "id": "ruleset_1_accept_payment",
+      "enabled": true,
+      "path": "static/rules.json"
```

Comment:
Not sure if this is the right place for the file. I saw a different approach in another PR, but it wasn't merged.

Reply:
static/rules.json seems good to me

Reply:
well, I mean I guess you could more specific than "rules", but inside static is good

Reply:
Updated `rules.json` -> `acceptPaymentRules.json`